source/deploy/deployment-overview.md

```diff
@@ -96,7 +96,7 @@ Mattermost Enterprise Edition supports Microsoft Active Directory and LDAP Singl
 
 Mattermost offers complete access to its Web Service APIs, along with incoming and outgoing webhooks, and slash command options for integrating with your self-managed systems.
 
-[Visit our app directory](https://about.mattermost.com/default-app-directory/) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and SVN, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
+[Visit our app directory](https://mattermost.com/marketplace/#privateApps) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and SVN, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
```

Comment:
Could you confirm where the "#privateApps" part comes from? It doesn't look like it needs to be there but would like to confirm.

Reply:
When you hit this URL https://about.mattermost.com/default-app-directory/, kindly check on the address bar. The URL gets resolved to https://mattermost.com/marketplace/#privateApps.

Reply:
Thanks for confirming! That makes sense.